examples/chef/common/chef-channel-manager.h

```diff
@@ -0,0 +1,64 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app/clusters/channel-server/channel-server.h>
+#include <vector>
+
+using namespace chip;
```

Comment:
do not place using in header files

Reply:
I think you mean not doing "using namespace" in headers that may expose whole namespace to the files including the header, right?  On the other hand, I can still use "using" a single type (e.g. using chip::CharSpan;), right?

Reply:
All using in the header file are removed.